ambari/datadog_checks/ambari/ambari.py

```diff
@@ -0,0 +1,198 @@
+                continue
+            if component.get(METRICS_FIELD) is None:
+                self.log.warning(
+                    "No metrics found for component {} for service {}"
+                        .format(component_name, service)
+                )
+                continue
+
+            for header in metric_whitelist:
+                if header not in component.get(METRICS_FIELD):
+                    self.log.warning(
+                        "No {} metrics found for component {} for service {}"
+                            .format(header, component_name, service)
+                    )
+                    continue
+
+                metrics = self.flatten_service_metrics(component.get(METRICS_FIELD)[header], header)
+                component_tag = COMPONENT_TAG + component_name.lower()
+                for metric_name, value in iteritems(metrics):
+                    metric_tags = base_tags + [component_tag]
+                    if isinstance(value, float):
+                        self._submit_gauge(metric_name, value, metric_tags)
+                    else:
+                        self.log.warning("Expected a float for {}, received {}".format(metric_name, value))
+
+    def _get_hosts_info(self, base_url, authentication, cluster):
+        hosts_endpoint = common.HOST_METRICS_URL.format(
+            base_url=base_url,
+            cluster_name=cluster
+        )
+        resp = self._make_request(hosts_endpoint, authentication)
+
+        return resp.get('items')
+
+    def _get_service_checks_info(self, base_url, authentication, cluster, service, service_tags):
+        service_check_endpoint = common.create_endpoint(base_url, cluster, service, SERVICE_INFO_QUERY)
+        service_info = []
+        service_resp = self._make_request(service_check_endpoint, authentication)
+        if service_resp is None:
+            service_info.append({'state': self.CRITICAL, 'tags': service_tags})
+            self.log.warning("No response received for service {}".format(service))
+        else:
+            state = service_resp.get('ServiceInfo').get('state')
+            service_info.append({'state': common.STATUS[state], 'tags': service_tags})
+        return service_info
+
+    def _make_request(self, url, auth):
+        try:
+            resp = self.http.get(url,
+                                 auth=HTTPBasicAuth(auth.get('username'), auth.get('password')),
+                                 verify=False)  # In case Ambari is under uncertified https
+            return resp.json()
+        except (requests.exceptions.HTTPError, requests.exceptions.ConnectionError) as e:
+            self.warning(
+                "Couldn't connect to URL: {} with exception: {}. Please verify the address is reachable"
+                .format(url, e))
+        except requests.exceptions.Timeout:
+            self.warning("Connection timeout when connecting to {}".format(url))
```

Comment:
I think you want `self.log.warning` for those last 2 calls.

Reply:
`self.warning` actually logs at warning level for you and makes your message appear on the status output - https://github.com/DataDog/integrations-core/blob/master/datadog_checks_base/datadog_checks/base/checks/base.py#L404

Reply:
Ah thanks I thought it was using the python warnings framework.